pkgs/applications/science/astronomy/gprojector/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv
+, lib
+, fetchzip
+, jre
+, makeDesktopItem
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gprojector";
+  version = "3.0.2";
+
+  src = fetchzip {
+    url = "https://www.giss.nasa.gov/tools/gprojector/download/G.ProjectorJ-${version}.tgz";
+    sha256 = "sha256-IvGZOYt2d8aWtlAJJzVrwkqOOhaUHUmEDlMeD/0NdwU=";
+  };
+
+  desktopItem = makeDesktopItem {
+    name = "gprojector";
+    exec = "gprojector";
+    desktopName = "G.Projector";
+    comment = meta.description;
+    categories = "Science;";
+    extraEntries = "StartupWMClass = gov-nasa-giss-projector-GProjector";
+  };
+
+  buildInputs = [ jre ];
+  nativeBuildInputs = [ makeWrapper desktopItem ];
```

Comment:
```suggestion
  nativeBuildInputs = [ makeWrapper copyDesktopItems ];
```

Reply:
I didn't know about copyDesktopItems for far, thanks for the suggestion :)